install/updateVersion.sh

```diff
@@ -66,6 +68,10 @@ if [[ -n ${CA_HUB_TAG} ]]; then
     CA_TAG="$(echo ${CA_HUB_TAG}|cut -f2 -d,)"
 fi
 
+if [[ -n ${ISTIO_NAMESPACE} ]]; then
+    ISTIO_NAMESPACE="$(echo ${ISTIO_NAMESPACE}|cut -f1 -d,)"
```

Comment:
don't copy paste, you don't need this (or if you did comment why) as there is no -n N1,N2 just -n N

replace with a default value though maybe